app/views/application/degree/index.njk

```diff
@@ -1,6 +1,6 @@
 {% extends "_form.njk" %}
 
-{% set title = "Degree" %}
+{% set title = "Add degree" %}
```

Comment:
I think this needs some logic around it @paulrobertlloyd

Reply:
Done!